internal/venice-common/src/main/java/com/linkedin/venice/guid/GuidUtils.java

```diff
@@ -31,25 +34,17 @@ private static GuidGenerator getGuidGenerator(VeniceProperties properties) {
     if (implName.equals(JavaUtilGuidV4Generator.class.getName())) {
       guidGenerator = new JavaUtilGuidV4Generator();
     } else if (implName.equals(DeterministicGuidGenerator.class.getName())) {
-      String jobId = properties.getString(ConfigKeys.PUSH_JOB_COMPUTE_JOB_ID, (String) null);
-      long guidGeneratorPrefix;
-      if (jobId == null) {
-        guidGeneratorPrefix = 0;
-      } else {
-        guidGeneratorPrefix = jobId.hashCode();
-      }
-
       guidGenerator = new DeterministicGuidGenerator(
-          guidGeneratorPrefix,
-          properties.getLong(ConfigKeys.PUSH_JOB_COMPUTE_TASK_ID, 0L));
+          properties.getLong(PUSH_JOB_GUID_MOST_SIGNIFICANT_BITS, 0L),
+          properties.getLong(PUSH_JOB_GUID_LEAST_SIGNIFICANT_BITS, 0L));
```

Comment:
Using `0L` as a default seems wrong... I think we should fail instead if the required config is not found.

Reply:
That makes sense. I'll update it. Probably also need to update some tests